src/style.css

```diff
@@ -1,3 +1,74 @@
+:root {
+  --main-color: #000;
+  --hover-color: #00acdc;
+}
+
 body {
   margin: 0;
+  font-family: "Roboto", sans-serif;
+  color: var(--main-color);
+}
+
+img {
+  display: block;
+  width: 100%;
+  object-fit: cover;
+  height: auto;
+}
```

Comment:
can you please provide class for img? we might have a lot of images on our page - we might don't want them to have exactly same styles

Reply:
Done.